src/styles/styles.scss

```diff
@@ -10,6 +10,8 @@ html {
     color: $color-white;
     font-family: Aeonik;
     min-height: 100vh;
+    padding-left: 4rem;
```

Comment:
I hope you are testing all the pages with a change like this.

This kind of change should not be in this PR.  But lets leave it this time, as long as you have tested.